tests/test_api.py

```diff
@@ -38,3 +82,111 @@ def test_get_setup_messages(self):
     def test_get_setup_messages_invalid_course_id(self):
         setup_messages = get_setup_messages('course-v1:edx+fake+19')
         self.assertIsNone(setup_messages)
+
+
+@ddt.ddt
+class GetBlockContentAPITests(TestCase):
+    """
+    Test suite for the get_block_content api function.
+    """
+
+    def setUp(self):
+        self.children = [
+            FakeChild('html', '01', '''
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                            Vivamus dapibus elit lacus, at vehicula arcu vehicula in.
+                            In id felis arcu. Maecenas elit quam, volutpat cursus pharetra vel, tempor at lorem.
+                            Fusce luctus orci quis tempor aliquet.
+                        </p>'''),
+            FakeChild('html', '02', '''
+                        <Everything on the content on this child is inside a tag, so parsing it would return almost>
+                            Nothing
+                        </The quick brown fox jumps over the lazy dog>'''),
+            FakeChild('video', '03'),
+            FakeChild('unknown', '04')
+        ]
+        self.block = FakeBlock(self.children)
+
+        self.course_id = 'course-v1:edx+test+23'
+
+    @ddt.data(
+        ('video', True),
```

Comment:
Should we include `html` here for completeness?

Reply:
Yes, thanks for catching that!